src/lucky/assignable.cr

```diff
@@ -40,6 +53,7 @@ module Lucky::Assignable
         end
       {% end %}
 
+      
```

Comment:
Looks like the formatter failed. This extra line probably has to go.

Reply:
Done